demisto_sdk/commands/doc_reviewer/tests/doc_reviewer_test.py

```diff
@@ -827,6 +828,33 @@ def test_find_known_words_from_pack_ignore_scripts_name(repo):
         assert script2.name in found_known_words
 
 
+def test_find_known_words_from_pack_ignore_commons_scripts_name(repo):
+    """
+    Given:
+        - Pack's structure is correct and pack-ignore file is present.
+        - The scripts are in the old version (JS code), no Scripts' dir exists (only yml amd md files).
+
+    When:
+        - Running DocReviewer.find_known_words_from_pack.
+
+    Then:
+        - Ensure the found path result is appropriate.
+        - Ensure the scripts names are ignored.
+    """
+
+    pack = repo.create_pack('test_pack')
+    script1_name = 'script-first_script'
+    pack._create_yaml_based(name=script1_name, dir_path=f'{pack.path}//Scripts', content={'name': script1_name})
+    script2 = pack.create_script(name='second_script')
+    rn_file = pack.create_release_notes(version='1_0_0', content=f'{script1_name}\n{script2.name}')
+    doc_reviewer = DocReviewer(file_paths=[])
+
+    with ChangeCWD(repo.path):
+        found_known_words = doc_reviewer.find_known_words_from_pack(rn_file.path)[1]
+        assert script1_name in found_known_words
+        assert script2.name in found_known_words
```

Comment:
lets choose one either script_name or script.name

Reply:
This is not the same. Script1 has no name as an attribute. Script2 has name and it is better using it.